applications/GeoMechanicsApplication/custom_elements/transient_Pw_interface_element.cpp

```diff
@@ -581,12 +581,12 @@ void TransientPwInterfaceElement<TDim, TNumNodes>::CalculateAndAddCompressibilit
 {
     KRATOS_TRY;
 
-    noalias(rVariables.PPMatrix) =
-        -PORE_PRESSURE_SIGN_FACTOR * rVariables.DtPressureCoefficient * rVariables.BiotModulusInverse *
-        outer_prod(rVariables.Np, rVariables.Np) * rVariables.JointWidth * rVariables.IntegrationCoefficient;
-
+    noalias(rVariables.PPMatrix) = GeoTransportEquationUtilities::CalculateCompressibilityMatrix(
+        rVariables.Np, rVariables.BiotModulusInverse, rVariables.IntegrationCoefficient,
+        rVariables.DtPressureCoefficient);
+ 
     // Distribute compressibility block matrix into the elemental matrix
-    rLeftHandSideMatrix += rVariables.PPMatrix;
+    rLeftHandSideMatrix += rVariables.PPMatrix * rVariables.JointWidth;
```

Comment:
Perhaps for clarity, we could add a pair of parentheses (to express that we only want to multiply `rVariables.PPMatrix`, not `rLeftHandSideMatrix`):
```suggestion
    rLeftHandSideMatrix += (rVariables.PPMatrix * rVariables.JointWidth);
```

Reply:
Added the parentheses.